style.css

```diff
@@ -0,0 +1,297 @@
+* {
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+  }
+  body {
+    font-family: sans-serif;
+    overflow: hidden;
+  }
+
+  :root{
+    --black-color: black;
+    --bg-color-1:#221C69;
+    --bg-color-2:#080942;
+    --bg-color-3:#431C76;
+    --bg-color-4:#663B94;
+  }
+  
+
+  .container {
+    max-height: 100vh;
+    overflow-y: scroll;
+    border: 1px solid gray;
+    scroll-snap-type: y mandatory;
+  }
+  .scroll-item {
+    height: 100vh;
+    scroll-snap-align: start;
+    color: rgb(87, 28, 28);
+    text-align: center;
```

Comment:
try following a single color refrences dont mix rgb and hex

Reply:
Since these changes are in main styles i'll upadge them once merged. Thank you for PR